account_bank_statement_import_camt/camt.py

```diff
@@ -198,6 +182,7 @@ def parse_statement(self, ns, node):
         transaction_nodes = node.xpath('./ns:Ntry', namespaces={'ns': ns})
         for entry_node in transaction_nodes:
             transaction = statement.create_transaction()
+            transaction.data = etree.tostring(entry_node)
```

Comment:
Yes it is. I decided to remove it from the parsing, as the transaction.data is filled with the element string contents as is, therefore not parsed.